### PR TITLE
feat(ca): move latencies media.tx/rx instead of MQE

### PR DESCRIPTION
**Pull request by torgeadelin**
## This pull request addresses
- move audio video setup time latency from MQE to media.tx/rx

image

### Change Type

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation update
- [ ] Tooling change
- [ ] Internal code refactor

## The following scenarios where tested
-unit test, manual test

### I certified that

- [x] I have read and followed [contributing guidelines](https://github.com/webex/webex-js-sdk/blob/master/CONTRIBUTING.md#submitting-a-pull-request)
- [x] I discussed changes with code owners prior to submitting this pull request

- [x] I have not skipped any automated checks
- [x] All existing and new tests passed
- [ ] I have updated the documentation accordingly

---

Make sure to have followed the [contributing guidelines](https://github.com/webex/webex-js-sdk/blob/master/CONTRIBUTING.md#submitting-a-pull-request) before submitting.

### CODE REVIEW

**Review by marcin-bazyl — APPROVED** [review]
looks good, I've added 1 small non-blocking suggestion in one of existing tests

File: packages/@webex/internal-plugin-metrics/test/unit/spec/call-diagnostic/call-diagnostic-metrics-batcher.ts [hunk 1]
```diff
@@ -342,10 +310,19 @@ describe('plugin-metrics', () => {
           );
         });
 
-        it('doesnt include audioSetup and videoSetup delays for other events', async () => {
+        it('appends the correct join times to the request for client.media-engine.ready', async () => {
+          webex.internal.newMetrics.callDiagnosticLatencies.getDiffBetweenTimestamps = sinon
+            .stub()
+            .returns(10);
```
[the remark this answers is not in the document]
Comment (torgeadelin) [reply packages/@webex/internal-plugin-metrics/test/unit/spec/call-diagnostic/call-diagnostic-metrics-batcher.ts:316]: added

**Review by Shreyas281299 — APPROVED** [review]
Change looks good. Just curious, this is marked as a bug fix, what was the issue that required these changes.

**Comment by torgeadelin** [comment]
> Change looks good. Just curious this is marked as a bug fix, what was the issue that required these changes.

related to this: https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-525195

**Pull request merged**